apps/arweave/src/ar_blacklist_middleware.erl

```diff
@@ -45,7 +45,6 @@ start() ->
 ban_peer(Peer, TTLSeconds) ->
 	Key = {ban, peer_to_ip_addr(Peer)},
 	Expires = os:system_time(seconds) + TTLSeconds,
-	ar_events:send(peer, {banned, Peer}),
```

Comment:
Did you have an intention to replace it with a direct ar_peers cast?

Reply:
ar_peers now calls `ar_blacklist_middleware:ban_peer` directly: https://github.com/ArweaveTeam/arweave/pull/443/files#diff-de37132d068e49cb11eb29b5c880b8ef2454d93f79686171b6cff9392824a2c5R353

We only ban peers in response to certain types of invalid blocks. Previously ar_block_pre_validator would both call `ban_peer` and send a `block, rejected` event. Now it just sends the event, which ar_peers handles and decides whether or not to ban the peer.